.devcontainer/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Ovsrpro Centos 7 Development Container" \
+  name="ovsrpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
+    make \
+    wget; \
+  # yum --assumeyes clean all; \
+  ln -s cmake3 /usr/bin/cmake; \
+  ln -s ctest3 /usr/bin/ctest; \
+  ln -s cpack3 /usr/bin/cpack; \
+  yum install --assumeyes \
+    # Required by GLEW
+    libX11-devel \
+    mesa-libGL-devel \
+    mesa-libGLU-devel \
+    # Required for Qt5
+    bison \
+    expat-devel \
+    fontconfig-devel \
+    freetype-devel \
+    gperf \
+    libxcb-devel \
+    libxkbcommon-devel \
+    openssl-devel \
+    which \
+    zlib-devel \
+    # Required by ZooKeeper
+    ant \
+    autoconf \
+    automake \
+    java-11-openjdk-devel \
+    libtool; \
```

Comment:
do you not need these

- ncurses-devel
- ninja-build
- glibs
- openssl
- dbus
- libXScrnSaver 
- libXtst
- pciutils
- EGL
- javac
I thought those were all necessary. but perhaps some of those are necessary only for developing applications that depend on ovsrpro. ?

Reply:
Nope. ncurses-devel, openssl, dbus, libXScrnSave, libXtst, pciutils, and EGL are not needed. Building ovsrpro does not work with Ninja, see #72.

glibs-devel might have solved the Qt build error, but I didn't know about that. I tried glibc-devel and glib-devel, neither of which worked. So I just removed `-glib` in #71